README.md

```diff
@@ -49,7 +49,7 @@ CoinSwap is a rust implementation of a variant of atomic-swap protocol, using HT
 Macos (M series) needs some feature for compilation 
 
 ```console
-$ cargo build --target=aarch64-apple-darwin  --features=build-macos-aarch
+$ cargo build --features=build-macos-aarch
```

Comment:
I don't know when we removed it, but we need to add the regular build command too in the readme.

The readme should have:
 - Command to build in regular Linux.
 - Note: Command for building on Mac.
 - Command to test in regular Linux.
 - Note: Command for testing on Mac.

Reply:
-  Will update the docs